packages/react/src/auto/polaris/PolarisAutoTable.tsx

```diff
@@ -93,13 +95,24 @@ export const PolarisAutoTable = <
         onQueryChange={search.set}
         onQueryClear={search.clear}
```

Comment:
Can you add `disabled={!!error}` to the `<IndexFilters>` so that it's impossible to search with an active error? Otherwise searching would still send API requests that error